### PR TITLE
added full screen gallery interstitials

### PR DESCRIPTION
**Pull request by rustytanton-zz**
please to review, thx

### CODE REVIEW

File: src/main/webapp/assets/styles/components/bsp-gallery-fullscreen.less [hunk 1]
```diff
@@ -217,3 +260,21 @@ body.bsp-gallery-fullscreen {
     }
 
 }
+
```
Comment (trhoppe) [inline src/main/webapp/assets/styles/components/bsp-gallery-fullscreen.less]: Can you put the below CSS here: https://github.com/perfectsense/brightspot-base/blob/master/src/main/webapp/assets/styles/components/move-to-bsp-carousel.less

File: styleguide/components/bsp-gallery-fullscreen/interstitial1.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<!-- @todo: will need to get some definition on types of content, sizes, etc to account for -->
+<div style="width: 560px; margin: 0 auto; padding-top: 100px;">
```
Comment (trhoppe) [inline styleguide/components/bsp-gallery-fullscreen/interstitial1.html]: Can you replace this with a base interstitial slide CSS rule for display: flex and to align and center vertically and horizontally whatever is in there? Then we can remove the wrapper and just have the iframe in this ajax response

File: src/main/webapp/assets/scripts/plugins/bsp-carousel-gallery.js [hunk 3]
```diff
@@ -91,11 +94,86 @@ export default {
...
+                    } else {
+                        $counter.empty().removeClass('counting');
+                        stage.enableNav();
+                        stage.showNav();
+                    }
+                }, 1000);
+            }
+            $content.html(response);
+        });
+        $el.data('destroy', () => {
```
Comment (trhoppe) [inline src/main/webapp/assets/scripts/plugins/bsp-carousel-gallery.js:165]: Any reason here why we can't just native slick to remove a slide? And by "here" I mean where this API is actually being used

Comment (rustytanton-zz) [inline src/main/webapp/assets/scripts/plugins/bsp-carousel-gallery.js:165]: We could, but then the user wouldn't see the interstitial again if they went back. This way they do.

**Pull request merged**